### DISCUSSION TITLE
Do you have any plans to deploy and maintain a DevLake instance for all ASF projects on GitHub?

### DISCUSSION THREAD

**Discussion by visortelle** [apache/devlake#5324] [Q&A]
Would like to have such a dashboard for Apache Pulsar. 🙂

**Comment by hezyin** [comment]
Hi @visortelle, I'd love to set something up for all ASF projects at some time, but may not be able to get to it in the short term.

For our future reference, what metrics/charts would you like to set up for Pulsar?

**Reply by visortelle** [reply]
Hi @hezyin,

> I'd love to set something up for all ASF projects at some time

That would be awesome 🙂
Until then, we'll probably experiment with a self-hosted instance and will provide some feedback.

> For our future reference, what metrics/charts would you like to set up for Pulsar?

Any metrics that can show community growth and project activity. Contributors count over time, PRs count, resolved issues count/ratio.
I'll be able to provide a more detailed answer after playing with it a bit.

**Reply by hezyin** [reply]
Thanks @visortelle, would love to get your feedback on DevLake!
